src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig

```diff
@@ -45,18 +46,25 @@
           <th>
             <p>{{ 'Qty'|trans({}, 'Admin.Orderscustomers.Feature') }}</p>
```

Comment:
Same as in previous comment: it could be nice to have it in full letters if we have enough space.

Reply:
done !